zakaria-touzri.txt

```diff
@@ -0,0 +1,190 @@
+https://codepen.io/Zakz/pen/ExaaxgZ?editors=1000
+
+<!-- reveiw questions to Submit in today's file firt-lastname.txt --> 
+<!--
+Copy the questions below into the firt-lastname.txt file on your github repo - answer each question and then push your changes. 
+
+    1. What is Semantic HTML? 
+
+Html written in a way that explains what is what ex:<footer> for the footer and <section> for a section
+
+    2. What is HTML used for? 
+
+It is used for the structure of a website and its content
+
+    3. What is an attribute and where do we put it? 
+
+an attribute is something that we use to call one element or a group of elements to make it easier (hopefully) the css part.
+
+    4. What is the h1 tag used for? How many times should I use it on a page?
+
+h1 is used for the visible title of the page and it is used only one time because it is better for SEO
+
+    5. Name two tags that have required attributes
+
+section and h2
```

Comment:
Section and h2 do not have required attributes. 
`<a>` however needs the `href` attribute and `<img>` needs the `src` attribute to function.